json/tokentype_string.go

```diff
@@ -4,6 +4,24 @@ package json
 
 import "strconv"
 
+func _() {
```

Comment:
This was added back in March 2019 to stringer in https://go-review.googlesource.com/c/tools/+/163637 which implies we have just not run stringer ever since it released the first stable version (`v0.1.0` was released in January 2021).

I avoided pinning to some unstable old version just for that purpose and picked the latest compatible stable version.